daffodil-cli/bin.NOTICE

```diff
@@ -133,29 +151,88 @@ Scala (lib/org.scala-lang.scala-library-<VERSION>.jar)
   and can be found in:
     daffodil-lib/src/main/scala/org/apache/daffodil/util/UniquenessCache.scala
 
-Scala Parser Combinators (lib/org.scala-lang.modules.scala-parser-combinators_<VERSION>.jar)
-  Scala parser combinators
+Scala Reflect (lib/org.scala-lang.scala-reflect-<REFLECT>.jar)
+  Scala
   Copyright (c) 2002-2021 EPFL
   Copyright (c) 2011-2021 Lightbend, Inc.
 
   Scala includes software developed at
   LAMP/EPFL (https://lamp.epfl.ch/) and
   Lightbend, Inc. (https://www.lightbend.com/).
 
-Scala XML (lib/org.scala-lang.modules.scala-xml_<VERSION>.jar)
-  scala-xml
-  Copyright (c) 2002-2020 EPFL
-  Copyright (c) 2011-2020 Lightbend, Inc.
+XML Resolver data (lib/org.xmlresolver.xmlresolver-<VERSION>-data.jar)
+  Schemas and documents from the W3C are being provided by the copyright
+  holders under the following license.
 
-  scala-xml includes software developed at
-  LAMP/EPFL (https://lamp.epfl.ch/) and
-  Lightbend, Inc. (https://www.lightbend.com/).
+  License
```

Comment:
I don't think this is NOTICE content, this is the actual [W3C license](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document), it's just confusing since that license uses the word NOTICE. I think this belongs in the LICENSE.bin file.

Reply:
Oh, you're right.  I saw the path `org/xmlresolver/notices/w3c-license.txt` inside the jar and thought it was a NOTICE due to the folder's name.  I've just pushed another commit removing the license text from here and adding the jar's name to the list of files under the W3C license in bin.LICENSE file.  With that change made, do you approve continuing to list all the jars here (even jars with no NOTICEs)?